filename.rb

```diff
@@ -2,6 +2,6 @@
 
 class Feature
  def do_thing
-
+   or_to_not_do_thing
```

Comment:
why would it or do nothing?

Reply:
Because maybe it does not want to do anything.